File: ResourceManager/ViewModels/Passwords/PasswordPageViewModel.cs
using Microsoft.Extensions.DependencyInjection;
using NetCore.Security;
using ResourceManager.Configuration;
using ResourceManager.Helpers;
using System;
using System.Collections.ObjectModel;
using System.Linq;

namespace ResourceManager.ViewModels
{
    public class PasswordPageViewModel : PageViewModel
    {
        public PasswordPageViewModel()
        {
            PageTitle = "Password";
            PageIcon = AssetManager.Instance.GetImage("Passwords.png");
            PageColor = Constants.PasswordPageColor;
            HasPermission = AuthManager.User.IsInPermission(Constants.VIEW_PASSWORD_PERMISSION);

            var tagsContent = App.ServiceProvider.GetRequiredService<TagsContentViewModel>();
            tagsContent.PageColor= PageColor;
            tagsContent.PasswordColumnWidth = 120;

            MenuItems = new ObservableCollection<PageContentViewModel>
            {
                App.ServiceProvider.GetRequiredService<PasswordContentViewModel>(),
                tagsContent
            };
            
            SelectedItem = MenuItems.FirstOrDefault();
            
        }
    }
}

File: ResourceManager/ViewModels/Passwords/PasswordPageViewModel.vb
Imports VBNetCore.Security
Imports ResourceManager.Configuration
Imports ResourceManager.Helpers
Imports System.Collections.ObjectModel
Imports System.Linq
Imports Microsoft.Extensions.DependencyInjection

Namespace ResourceManager.ViewModels
    Public Class PasswordPageViewModel
        Inherits PageViewModel
        Public Sub New()
            PageTitle = "Password"
            PageIcon = AssetManager.Instance.GetImage("Passwords.png")
            PageColor = PasswordPageColor
            MyBase.HasPermission = AuthManager.User.IsInPermission(VIEW_PASSWORD_PERMISSION)

            Dim tagsContent = Application.ServiceProvider.GetRequiredService(Of TagsContentViewModel)()
            tagsContent.PageColor = PageColor
            tagsContent.PasswordColumnWidth = 120

            MenuItems = New ObservableCollection(Of PageContentViewModel) From {
                Application.ServiceProvider.GetRequiredService(Of PasswordContentViewModel)(),
                tagsContent
            }

            SelectedItem = MenuItems.FirstOrDefault()

        End Sub
    End Class
End Namespace
